src/mapping/query/functions.rs

```diff
@@ -190,12 +190,14 @@ impl Function for ToTimestampFn {
 }
 
 fn to_timestamp(value: Value) -> Result<Value> {
-    use chrono::{DateTime, TimeZone, Utc};
+    use chrono::{TimeZone, Utc};
 
     match value {
-        Value::Bytes(b) => DateTime::parse_from_rfc3339(&String::from_utf8_lossy(&b))
-            .map(|dt| dt.with_timezone(&Utc))
-            .map_err(|_| "cannot parse string as RFC3339".to_owned()),
+        Value::Bytes(b) => {
+            return Conversion::Timestamp
+                .convert(Value::Bytes(b))
+                .map_err(|e| e.to_string())
+        }
```

Comment:
Does this work?
```suggestion
        Value::Bytes(b) => Conversion::Timestamp
                .convert(value)
                .map_err(|e| e.to_string()),
```

Reply:
It doesn't, because `convert` returns a `Result<Value>`, whereas the other arms return `Result<DateTime>`.

Reply:
Oh, I see, I missed that in the reduced context of the diff.

Given the other two arms that return a value are pretty simple, I think removing the map would simplify things here.